### ISSUE TITLE
Ins manual for Wasted configuration

### ISSUE THREAD

**Issue by 19871947** [x13a/Wasted#55]
Hello Guys,
We are a group of friends interested to use the app, but we are having problems with the functions.
Is there anywhere an instructions manual how to use the app? Some specs are too complicated for the normal user.

1. Can we get some explanations on the following: Tile function and tile time delay. How does the function work? the safe delay is represented as 0.5, 1.0, 1.5 ...- these are  minutes or its half hour and hour etc? How can a wipe be canceled in the safe delay time period?

2. Panic Kit- desc reads "collection of tools for creating panic buttons"...- This explains all, but where do we configure the panic buttons and reactions?

3. On the main screen what are the color representations? Sometimes the sec code is yellow sometimes its red? Do wipe data has to be on for the settings to fire or its a function to wipe it momentarily?

4. What is the Shortcut option? - desc reads  _"It is a button you will see when you make a long tap"_. The button fires or what is the point of it?

Recast is also way too difficult to understand for the normal everyday user.

I think the idea behind the app is amazing, and it's also great that someone bothered to create it, we just might use a little manual so it could be used by widespread public.
Thanks

**Comment by Tunga2329** [comment]
Excelentes preguntas... un manual de uso seria genial 😁

**Comment by x13a** [comment]
> Tile function and tile time delay. How does the function work?

_Tile_ is a button in quick settings menu. It mimics to **airplane mode**. **Delay** is in seconds. You have to click a tile again to cancel fire before a delay will out.

> where do we configure the panic buttons and reactions?

_PanicKit_ requires another app to trigger Wasted. This app has to implement _PanicKit_ support. Example is **Ripple**.

> what are the color representations?

Color change is historical. It changes to yellow when any trigger is checked.

> Do "wipe data" checkbox ..

When _wipe data_ is check on fire Wasted will factory reset your device, else it will only lock it. Without bottom ON it won't do anything.

> What is the Shortcut option?

If you will check _Shortcut_ and turn on Wasted you will see a panic button when you make a long tap on the Wasted icon.

> Recast is also way too difficult to understand ..

_Recast_ is to send a broadcast message on fire. It gives an ability to run some another app after Wasted locks a device.

TLDR.  
**Most of these have already discussed here in issues.**
